pkg/util/paramtable/quota_param.go

```diff
@@ -364,10 +364,10 @@ The maximum rate will not be greater than ` + "max" + `.`,
 			rate := megaBytes2Bytes(getAsFloat(v))
 			// [0, inf)
 			if rate < 0 {
-				return min
+				return p.DMLMinInsertRate.GetValue()
```

Comment:
If the user sets a global minimum rate, it may not be properly aligned with the collection minimum rate due to being beyond the user's expectations.